Cargo.toml

```diff
@@ -206,6 +206,11 @@ lto = true
 codegen-units = 1
 debug = "full"
 
+[profile.dev]
```

Comment:
there is this porfile used, in development.
As if so, I would with we expecialy need all of these things

Reply:
`dev` profile is already optimized for speed, I could have created a custom profile from it but thought that having `debug=0` could be the default.

Reply:
it makes the linking stage considerably faster